garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -1047,3 +999,60 @@ L.set_cross_size = "Fadenkreuz Größe"
 
 -- 5-25-15
 L.hat_retrieve = "Du hast den Hut eines Detektivs aufgehoben."
+
+-- ?-??-16
+L.item_radar_desc = [[
+ Dies erlaubt dir Lebenszeichen zu scannen.
+ Es startet automatisch beim Kauf.
+ Stelle es im Radar Reiter dieses Menüs ein.]]
+ 
+L.item_disg_desc  = [[
+ Versteckt deine ID. Vermeidet außerdem, dass Du die
+ Person bist, die als letztes vom Opfer gesehen wurde.
+ 
+ Schalte es im Tarnungs Reiter ein oder aus
+ oder drücke Enter auf dem Numpad.]]
+
+L.decoy_desc      = [[
+ Zeigt Detektiven ein gefälschtes Radar-Signal
+ und bewirkt, dass der DNA Scanner den
+ Ort der Attrappe zeigt, wenn sie nach deiner
+ DNA suchen.]]
+ 
+L.defuser_desc    = [[
+ Entschärft sofort eine C4 Bombe.
+ 
+ Unbegrenzt nutzbar. C4 wird leichter zu entdecken
+ sein, wenn du das bei dir trägst.]]
+ 
+L.flare_desc      = [[
+ Kann benutzt werden, um Leichen zu verbrennen,
+ damit sie nie gefunden werden können.
+ Begrenzte Munition.
+ Das Verbrennen einer Leiche macht ein ganz
+ bestimmtes Geräusch.]]
+ 
+L.hstation_desc   = [[
+ Ermöglicht bei Platzierung, dass sich jeder Spieler
+ an ihr heilen kann. Langsame Wiederaufladung,
+ kann beschädigt werden.
+ Am Gerät sind DNA Spuren ihrer Benutzer zu finden.]]
+
+L.sipistol_desc   = [[
+ Handfeuerwaffe mit geringer Lautstärke,
+ nutzt normale Pistolenmunition.
+ 
+ Opfer schreien nicht, wenn sie damit getötet werden.]]
+ 
+L.binoc_desc      = [[
+ Zoom an eine Leiche ran um sie aus einer
+ großen Entfernung zu identifizieren.
+ 
+ Unbegrenzt nutzbar, aber das Identifizieren dauert
+ inige Sekunden.]]
```

Comment:
I guess an "e" got stripped. It should be _einige Sekunden_ and not _inige Sekunden_.